src/main/java/io/kestra/plugin/influxdb/InfluxQLQuery.java

```diff
@@ -0,0 +1,168 @@
+package io.kestra.plugin.influxdb;
+
+import com.influxdb.client.InfluxDBClient;
+import com.influxdb.client.InfluxQLQueryApi;
+import com.influxdb.query.InfluxQLQueryResult;
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.executions.metrics.Counter;
+import io.kestra.core.models.property.Property;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.models.tasks.common.FetchType;
+import io.kestra.core.runners.RunContext;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import java.io.File;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Query InfluxDB using InfluxQL",
+    description = "Execute an InfluxQL query against InfluxDB and return the results"
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "Execute an InfluxQL query and store results in execution",
```

Comment:
```suggestion
            title = "Execute an InfluxQL query",
```
I checked other plugins we don't really say that we store in exec, better to keep doc simple to avoid confusion